### ISSUE TITLE
Drop more platforms we don't support

### ISSUE THREAD

**Issue by sunweaver** [ArcticaProject/nx-libs#275]
Related X.org commits:

```
commit 5035741fd4cb4918957064a8c9ded9c7c31b4e22
Author: Adam Jackson <ajax@redhat.com>
Date:   Wed Jul 23 13:38:38 2008 -0400

    Unifdef __osf__

commit bd8bd2c700e6ccac53b47c45766c0a3a08a59344
Author: Adam Jackson <ajax@redhat.com>
Date:   Wed Jul 23 13:36:35 2008 -0400

    Unifdef AIX.

commit 5f5eed7f790db2f0ae0cd0f5c5ee5a312079c0b0
Author: Adam Jackson <ajax@redhat.com>
Date:   Wed Jul 23 13:27:24 2008 -0400

    Unifdef DGUX.

commit 856db05b58b71c5ff22af727aba435a8c356abfa
Author: Adam Jackson <ajax@redhat.com>
Date:   Wed Jul 23 13:25:37 2008 -0400

    Unifdef sgi.

commit b699364b1c807d29666633523f5b5a608a07a8ee
Author: Adam Jackson <ajax@redhat.com>
Date:   Wed Jul 23 11:43:59 2008 -0400

    Unifdef VXWORKS.

commit 0ff5bc404842f7264a8e77f9e2a0bd8d0b051356
Author: Adam Jackson <ajax@redhat.com>
Date:   Wed Jul 23 09:09:11 2008 -0400

    Unifdef QNX.
    
    Again, hasn't worked since at least 7.0.

```

Not sure, if we still have all of the above ifdefs in nx-libs code. If yes, they should all go.

**Comment by sunweaver** [comment]
VXWORKS is not present in nx-libs.

**Comment by sunweaver** [comment]
I have commits pending for this...

**Comment by sunweaver** [comment]
Let's consider #275 has resolved by 0a314adfa63669155299c77e479ed68c349643bd. For remaining ancient platforms, we have individual issues open.
